defaults/main.yml

```diff
@@ -5,6 +5,6 @@ deploy_group: '{{ ansible_ssh_user }}'
 laravel_rotate_nginx_log: true
 laravel_rotate_nginx_log_retention: 30
 
-laravel_https_enabled: no
+nginx_use_ssl: true
```

Comment:
You can keep the name `laravel_https_enabled`. We want to have laravel_ as namespace for variables.